### ISSUE TITLE
Why the loss at the beginning of training is very low?

### ISSUE THREAD

**Issue by JXMss** [morpheusthewhite/Faster-RCNN-TensorFlow-Python3.5#6]
I change the code to train whole(of coz I delete pretrained model and checkpoint file
but total loss fluctuates between 3 and 0.1, And it's the same after 80k steps.
Is this normal？

**Comment by morpheusthewhite** [comment]
Can you post a picture of the graph here?

**Comment by JXMss** [comment]
> Can you post a picture of the graph here?

Do u mean the graph of tensorboard？

**Comment by morpheusthewhite** [comment]
Yes

**Comment by JXMss** [comment]
Here u r
![TIM截图20200320211637](https://user-images.githubusercontent.com/40733256/77166970-209bd400-6af0-11ea-8fda-7b9446c687da.jpg)

**Comment by JXMss** [comment]
and here is the total loss
![image](https://user-images.githubusercontent.com/40733256/77167119-593bad80-6af0-11ea-8cc1-1e176d09c67f.png)

**Comment by JXMss** [comment]
This one maybe clearer
![image](https://user-images.githubusercontent.com/40733256/77167257-a1f36680-6af0-11ea-95af-d2ff1bae3f3a.png)

**Comment by JXMss** [comment]
![image](https://user-images.githubusercontent.com/40733256/77167356-dc5d0380-6af0-11ea-85fd-c41d7b749f14.png)

**Comment by JXMss** [comment]
Is there anything wrong?

**Comment by morpheusthewhite** [comment]
But how did you replace the pretrained net?

**Comment by JXMss** [comment]
I didn't remember all
But the main step is delete line 129 to 140 in train.py and line 54 to 91 in lib/nets/vgg16.py
And then fix or delete other error code to make the program run successfully

**Comment by JXMss** [comment]
I also changed the "trainable" and "is_training" parameter in vgg16.py

**Comment by morpheusthewhite** [comment]
Then it is most likely caused by the fact that your new net, without the pretrained model, is too simple to model the problem. Try adding new layers
